source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -83,5 +83,3 @@ To put this concept into practice, see the following resources:
 *	:ref:`telem-guide`
 *	`Telemetry feature description`_
 
```

Comment:
@tpleavitt  I agree about modifying the last sentence. Suggest something like: 
    To put this concept into practice, refer to :ref:`telem-guide`.

Also delete line 84 (2nd bullet), along with the link definition in lines 86-87.

Reply:
Sounds good

Reply:
@tpleavitt  please check for back-ticks around telem-guide? GH ate them in my response. 
Should be  "":ref:`telem-guide`""

Reply:
Oops!  Done